content/en/docs/getting-started/prerequisites/hardware/snp.md

```diff
@@ -8,5 +8,109 @@ tags:
 - SEV
 ---
 
-TODO
+## Platform Setup
+
+> [!WARNING]
+>
+> In order to launch SEV or SNP memory encrypted guests, the host must be prepared with a compatible kernel, `6.8.0-rc5-next-20240221-snp-host-cc2568386`. AMD custom changes and required components and repositories will eventually be upstreamed. 
+
+> [Sev-utils](https://github.com/amd/sev-utils/blob/coco-202402240000/docs/snp.md) is an easy way to install the required host kernel, but it will unnecessarily build AMD compatible guest kernel, OVMF, and QEMU components. The additional components can be used with the script utility to test launch and attest a base QEMU SNP guest. However, for the CoCo use case, they are already packaged and delivered with Kata.
```

Comment:
The information in this warning is really what the scope of this entire page should be. 

The prerequisites should not include starting up workloads. It should just tell the reader how to setup their machine.

So, what hw is supported in the first place; how to check that the host is valid; Firmware; Kernel, where to get it; etc

If you really want to have a full e2e flow for SNP, you could include that in the examples section, but I would avoid doing this because it will require more maintenance.

Reply:
@fitzthum I removed the warning, and I relabeled the steps to bringing it up as an example. I think having it there is important, because I refer to that when giving instructions on how to validate that the TEE environment is being used. Let me know if you things should be changed in that section.

Reply:
Prerequisites are for things that come before installing the operator. This page should not include any information about installing the operator or running a workload. If you would like to have an e2e example of using SNP, you can put that in the examples section. If you want to show people how to verify that they are inside an enclave, add a section after sample workload, that has a subsection for checking that you are in a SNP enclave (and could have pages for other platforms)